sim/src/service/engineService.test.cpp

```diff
@@ -272,20 +272,12 @@ TEST_F(EngineServiceTest, GetAndSetAttribute) {
     auto testSim = test_simulation::TestSimulation();
     applySim(testSim.simDef);
 
-    // Step the sim so that init graph is executed
-    StepSimulationReq stepReq;
-    StepSimulationResp stepResp;
-    ClientContext stepContext;
-    stepReq.set_name(testSim.SIM_NAME);
-    Status s = client->StepSimulation(&stepContext, stepReq, &stepResp);
-    ASSERT_TRUE(s.ok());
```

Comment:
Can we move this test to test that `ApplySimulation` runs the Init Graph instead of removing it entirely

Reply:
This code section is used for testing get and set attribute. Because the old code assumed that stepping is required for initGraph to run, hence, this code was here. It has now been removed because initGraph runs on apply, not on step.

So, now it is also implicitly testing that initGraph is run on apply.

Reply:
Ok, but there is no test that verifies that the Init Graph is executed (ie  setting a non 0/"" value using init Graph and checking that its set on apply). Can we add that?

Reply:
I have updated the starting value to `30` in 525d33a